app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
I talked to @svbergerem via IRC, and he meant that it is doable with normal effort to use the data already there in the frontend (you maybe need to convert the author objects from the post to another structure, but then can use it for typeahead as local data. But that's a simple mapping and no problem).

@cmrd-senya: so there is no need to change the controller here and the check for the `q` param presence is OK. If you need help for doing this in the frontend, it is possibly the best to talk to @svbergerem directly.